Wendy/assignment1Answers.html

```diff
@@ -0,0 +1,219 @@
+				</div>
+			</div>
+			</div>
+			<div class="row" id="part2">
+			<h3> Part II </h3>	
+				<div class="col-md-12">
+					<ul>
+						<li> a.  var mean = (x + y + z)/3</li>
+						<li> b.  x= x + (mean -x)*.5></li>
+						<li> b.(cont) y = y + (mean - y)*.5></li>
+						<li> b. (cont) z = z + (mean - z)*.5></li>
+					</ul>
+				</div>
+			</div>
+			<div class="row" id="part3">
+			<h3> Part III </h3>
+				<div class="col-md-6">	
+				<h3> a.  Calculating Area of Rectangle</h3>
+				<ul>
+					<li> area = (t-b)*(r-l)</li>
+				</ul>
+				<h3> b.  True If the rectangle is taller than it is wide... </h3>
+				<ul>
+					<li> area for rectangle with taller than wide </li>
+					<li> (Math.abs(t-b))>(Math.abs(l-r))</li>
+				</ul>	
+				<h3> c.  Rectangle enclosing circle </h3>
+				<ul>	
+					<li> var circumference; </li>
+					<li> Step 1:  Determine length of the shortest side. </li>
+					<li> Step 2:  Multiply that number by Math.PI</li>
+					<li> (Math.abs((Math.abs(t-b))-(Math.abs(l-r))))*Math.PI; </li?
+				</ul>
+				</div>
+				<div class="col-md-6">
+				<h3> d.  The circle enclosing the rectangle</h3>
+				<ul>
+					<li> Step 1:  Determine the length of the longest side.</li>
+					<li> Step 2:  Multiply that number by Math.PI</li>
+					<li> ((l-r)<(t-b)&&((t-b)*Math.PI))||((t-b)<(l-r)&&((l-r)*Math.PI));</li>
+				</ul>
+				<h3> e.  Variables for expressing smaller rectangles</h3>
+				<ul>
+					<li> var areaSmall</li>
+					<li> 1. ((l-r)/3)*((t-b)/3) </li>
+					<li> var positionT</li>
+					<li> 2. (t-b)/3 + b + (t-b)/3;;</li>
+					<li> varSquare (boolean checking if the small rec is a square)
+					<li> 3. (Math.abs(t-b))===(Math.abs(r-l))</li>
+					<li> var positionR</li>
+					<li> 4. (l-r)/3 + r + (l-r)/3;
+
+					<!-- <li> var areaSmall (to represent the center small rectangle)</li>
+					<li> 1. areaSmall = (t/3)*(l/3)</li>
+					<li> var tallSmall (to represent the tallest side)</li>
+					<li> 2. var tallSmall = (l>=t)&&(l/3)||!(l>=t)&&(t/3)</li>
+					<li> var shortSmall (to reperesent the shorter side)</li>
+					<li> 3. var shortSmall = (l<=t)&&(l/3)||(t<=l)&&(t/3)</li>
+					<li> var square (boolean checking if the small rectangle is a square)
+					</li> 4. varSquare = ((t-l) === 0)</li> -->
+				</ul>
+				</div>
+			</div>	
+			
+			<div class="references">
+			<div class="row">
+			<h3>Resources used</h3>
+				<div class="col-md-12">
+				<ul>
+					<li><a href="http://www.w3schools.com/jsref/jsref_pi.asp"> Math.PI info</a></li>
+					<li><a href="http://www.w3schools.com/js/js_variables.asp"> Specific Definitions for Variables in JavaScript</a></li>
+				</div>
+			</div>
+			</div>		
+			<div class="row" id ="part4">
+			<h3>Part IV</h3>
+			<h3> Checker Board solving for Number, Row and Column</h3>
+				<div class="col-md-6">
+					
+					<ul>
+						<li> a.) n = ((r*7)+(r+c))</li>
+						<li> b.) r = ((n-8)%8)</li>						
```

Comment:
Looks like Dan wanted you to use Math.floor for this one, which makes it much less complex. This rounds the solution down the the nearest whole number. You would need to use this function, and then simply divide n by 8. Here is the solution: r = Math.floor(n/8);